_config/Oembed.yml

```diff
@@ -1,40 +0,0 @@
-name: Oembed
----
-Oembed:
```

Comment:
Presumably the 3rd party module comes pre-loaded with this configuration? If not, we'd want to pass it in.